superset/migrations/versions/2024-05-01_10-52_58d051681a3b_add_catalog_perm_to_tables.py

```diff
@@ -45,10 +49,21 @@ def upgrade():
         "slices",
         sa.Column("catalog_perm", sa.String(length=1000), nullable=True),
     )
+
+    if not table_has_index(perm_table, perm_index_view_menu):
```

Comment:
might be good to add these into a helper function call create_index that will do the check and add it like here https://github.com/apache/superset/pull/31303

Reply:
Nice, I was thinking of a helper function. I don't want to create a conflict with that PR. Perhaps we can use his helper function after it's merged.